### PR TITLE
Add content, mess around with sidebar links

### PR DESCRIPTION
**Pull request by abesto**
* Added quickstart instructions
 * Added list of existing instrumentation libraries from Zipkin README
    * Added `finagle-zipkin` to the list
 * Turned the GitHub, Twitter links in the sidebar into logos, added Gitter there

(Test site still available at http://abesto.github.io/openzipkin.github.io/)

screenshot 2016-02-29 23 39 22

### CODE REVIEW

**Comment by abesto** [comment]
@adriancole @rtyler Here's your daily dose of documentation :)

File: _data/existing_instrumentations.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+- language: Python
+  library: >-
```
Comment (adriancole) [inline _data/existing_instrumentations.yml]: nice! cc @prat0318

File: _data/existing_instrumentations.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- language: Java
+  library: >-
+    [brave](https://github.com/openzipkin/brave)
+  framework: Jersey, RestEASY, JAXRS2, Apache HttpClient, Mysql
+  transports: Http, Kafka, Scribe
+  sampling: "Yes"
+  notes: Java 7 or higher
+
+- language: Java
```
Comment (adriancole) [inline _data/existing_instrumentations.yml]: Scala

File: _data/existing_instrumentations.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    [pyramid_zipkin](https://github.com/Yelp/pyramid_zipkin)
+  framework: >-
+    [Pyramid](http://docs.pylonsproject.org/projects/pyramid/en/latest/)
+  transports: >-
+    [Kafka \\| Scribe](http://pyramid-zipkin.readthedocs.org/en/latest/configuring_zipkin.html#zipkin-transport-handler)
+  sampling: >-
+    [Yes](http://pyramid-zipkin.readthedocs.org/en/latest/configuring_zipkin.html#zipkin-tracing-percent)
+  notes: py2, py3 support.
+
+- language: Java
```
Comment (adriancole) [inline _data/existing_instrumentations.yml]: LGTM cc @kristofa

File: _data/existing_instrumentations.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  library: >-
+    [finagle-zipkin](https://github.com/twitter/finagle/tree/develop/finagle-zipkin) 
+  framework: >-
+    [Finagle](https://github.com/twitter/finagle)
+  transports: Scribe
+  sampling: "Yes"
+
+- language: Ruby
```
Comment (adriancole) [inline _data/existing_instrumentations.yml]: LGTM cc @jcarres-mdsol

File: _data/existing_instrumentations.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- language: "C#"
+  library: >-
+    [ZipkinTracerModule](https://github.com/mdsol/Medidata.ZipkinTracerModule) 
+  framework: OWIN, HttpHandler  
+  transports: Http
+  sampling: "Yes"
+  notes: lc support. 4.5.2 or higher
+
+- language: Go
```
Comment (adriancole) [inline _data/existing_instrumentations.yml]: LGTM cc @elodina

File: pages/existing_instrumentations.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+The following libraries exist to provide instrumentation on various platforms.
+Please refer to their individual documentation for setup and configuration
+guides.
+
+| Language | Library | Framework | Transports Supported | Sampling Supported? | Other notes |
+|:---------|:--------|:----------|:---------------------|:--------------------|:------------|{% for lib in site.data.existing_instrumentations %}
+| {{ lib.language }} | {{ lib.library }} | {{ lib.framework }} | {{ lib.transports }} | {{ lib.sampling }} | {{ lib.notes }} |{% endfor %}
+{: .wide-table}
+
+Did we miss a library? Please open a pull-request to
```
Comment (adriancole) [inline pages/existing_instrumentations.md]: nice

File: pages/instrumenting.md [hunk 7]
```diff
@@ -2,48 +2,18 @@
...
-
-Let's walk through the parts required to instrument a service that doens't
-already have support for Zipkin.
+This is an advanced topic. Before reading further, you may want to check whether
+an instrumentation library for your platform [already exists]({{ site.github.url
+}}/pages/existing_instrumentations).
+{: .message}
+
+So you want to send traces to Zipkin, and there is no existing library for your
+platform. If you want to take on creating an instrumentation library, here's
```
Comment (adriancole) [inline pages/instrumenting.md]: I can't think of a good way to rephrase this, so sounds good :P

**Comment by adriancole** [comment]
cool.. only one point about the language of finagle as scala

File: _data/existing_instrumentations.yml (hunk 4 above)
Comment (jcarres-mdsol) [inline _data/existing_instrumentations.yml]: seems accurate

File: _data/existing_instrumentations.yml (hunk 1 above)
Comment (prat0318) [inline _data/existing_instrumentations.yml]: lgtm

File: _data/existing_instrumentations.yml (hunk 3 above)
Comment (kristofa) [inline _data/existing_instrumentations.yml]: Nice, looks good!

**Comment by kristofa** [comment]
Nice update to the documentation! Well done!

**Pull request merged**